src/main/kotlin/ai/serenade/intellij/services/CommandHandler.kt

```diff
@@ -201,9 +202,10 @@ class CommandHandler(private val project: Project) {
         remainingCommands: List<Command>,
         read: () -> CallbackData?
     ) {
-        ApplicationManager.getApplication().invokeLater {
-            runCommandsInQueue(callback, remainingCommands, read())
-        }
+        ApplicationManager.getApplication().invokeLater(
+            { runCommandsInQueue(callback, remainingCommands, read()) },
+            ModalityState.any()
```

Comment:
Curious what this does but not blocking.

Reply:
By explicitly setting `ModalityState.any()`, this command will run regardless of whether there are open dialogs or not. I refactored this a bit so that we can explicitly set whether we want this to be the case.